Compentio.SourceMapper.Generator/Compentio.SourceMapper.csproj

```diff
@@ -51,4 +51,19 @@
     </AssemblyAttribute>
   </ItemGroup>
 
```

Comment:
Is it will be localized? As far as I understand resouces used for labels, texts (UI texts, sometimes exception texts, when need to be localized). 

Are assembly names planed to be changed or localized or used in few places in code?

Reply:
I really don't like magic numbers and magic strings. I often gather it in resx like AppStrings and not generate any other localization version of these strings. They can be used in source code and test code. If is it a bad behavior, can be reverted.

Reply:
Here it is not a number it is hardcoded library name. Placing it in resources actually means configuration entry. (resx files are not actually configurations - these are strings, texts, localized texts). IMO for string keys (like library name) resources gives nothing. If you do not like magic numbers (SonarCube) also does ot like - you can give a `private const` for it.